### PR TITLE
DEV-32015 Github actions to create unified manifest file

### PR DESCRIPTION
**Pull request by barakakoun**
latest realease updates in master merge

### CODE REVIEW

File: .github/workflows/update-unified-manifest.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+          python-version: 3.8
+      - name: Run script
+        uses: jannekem/run-python-script-action@v1
+        with:
+          script: |
+            import os, json
+            with open("manifest.json", "r+") as t:
+              t.truncate(0)
+              t.write('[')
+              first = True
```
Comment (ojamui) [inline .github/workflows/update-unified-manifest.yml:22]: didnt we find a json lib to parse this? i really think this is redundant and dangerous

File: .github/workflows/update-unified-manifest.yml [hunk 2]
```diff
@@ -0,0 +1,41 @@
+name: build unified manifest file
```
Comment (ojamui) [inline .github/workflows/update-unified-manifest.yml:1]: would change the name to build manifest or something simpler

**Comment by ojamui** [comment]
please add description showing the result and what u need to do to call it

File: .github/workflows/update-unified-manifest.yml (hunk 1 above)
Comment (barakakoun) [reply .github/workflows/update-unified-manifest.yml:22]: the only thing the json library is good for here is using `dumps` function to write the json to file, and it parse it with `/n`s ans `/t`s, so the normal text writer just works better...
couldn't find a solution specific enough merge multiple json files into an array, all the solutions i found simply run over the files and write them to the file one by one

Comment (ojamui) [reply .github/workflows/update-unified-manifest.yml]: hmm and something like this doesnt owrk?
image

**Comment by barakakoun** [comment]
retest this please

**Comment by barakakoun** [comment]
retest this pleae

**Comment by barakakoun** [comment]
retest this please

**Review by ShiranAvidov — APPROVED** [review]
lgtm

**Pull request merged**